desc/compute/_omnigenity.py

```diff
@@ -443,15 +443,25 @@ def _omni_map(params, transforms, profiles, data, **kwargs):
     iota = kwargs.get("iota", 1)
 
     # coordinate mapping matrix from (alpha,h) to (theta_B,zeta_B)
+    # need a bunch of wheres to avoid division by zero causing NaN in backward pass
+    # this is fine since the incorrect values get ignored later, except in OT or OH
+    # where fieldlines are exactly parallel to |B| contours, but this is a degenerate
+    # case of measure 0 so this kludge shouldn't affect things too much.
+    mat_01 = jnp.array(
+        [[N, iota / jnp.where(N == 0, 1, N)], [0, 1 / jnp.where(N == 0, 1, N)]]
+    )  # OP
+    mat_10 = jnp.array([[0, -1], [M, -1 / jnp.where(iota == 0, 1.0, iota)]])  # OT
+    den = jnp.where((N - M * iota) == 0, 1.0, (N - M * iota))
+    mat_11 = jnp.array([[N, M * iota / den], [M, M / den]])  # OH
```

Comment:
What's the naming convention of `mat_01` vs `_10` vs `_11`? I suggest renaming to `mat_OP`, `_OT`, `_OH` for clarity.

Reply:
this was meant to be like `mat_MN` where `MN in [01, 10, 11]`, but I like your idea better.